### ISSUE TITLE
"Cannot find NUCLEUS.DATA : DEAD"

### ISSUE THREAD

**Issue by robinhood2014** [ogoguel/nucleus-p8#2]
I just tried to boot up an Apple IIgs (really, an emulator) from the disk image, but the IIgs complains that it "Cannot find NUCLEUS.DATA : DEAD".

Also, when I try to execute NUCLEUS.SYSTEM (from BASIC.SYSTEM, since it can't run from GS/OS), the IIgs hangs on the "FTA Smartboot 3.0 Special AppleFest Version" screen.

What's the deal here?

**Comment by ogoguel** [comment]
* First error means that the NUCLEUS.DATA file cannot be located nearby the NUCLEUS.SYSTEM : can you share a screenshot of your boot disk 
* Seems like you are launching Basic.System from GS/OS : you need to launch Nucleus from a pure PRODOS8 environment. Let me know if it helps

**Comment by robinhood2014** [comment]
![GSplus v0 14   6_21_2022 7_24_01 PM](https://user-images.githubusercontent.com/6971466/174917786-cbda8286-34e8-424b-9a4c-a4e89e5ead5e.png)

**Comment by ogoguel-holoforge** [comment]
My mistake : not a screenshot of the app, but a screenshot of your mounted disks diretories (boot, data, ...), as well as details of your launch sequence (for example, booting on P8, launching basic.system, switching deive, executing nucleus.system,... ). 

Normally, the Nucleus.data file should be located nearby the nucleus.system file, but it fails obviously for you, hence the needed detail to better understand your file structure. 

Thanks for your time

**Comment by ogoguel** [comment]
My mistake : not a screenshot of the app, but a screenshot of your mounted disks diretories (boot, data, ...), as well as details of your launch sequence (for example, booting on P8, launching basic.system, switching deive, executing nucleus.system,... ). 

Normally, the Nucleus.data file should be located nearby the nucleus.system file, but it fails obviously for you, hence the needed detail to better understand your file structure. 

Thanks for your time

**Comment by robinhood2014** [comment]
Okay. I inserted the disk into the drive, turned on the IIgs, and booted into ProDOS 8, which started BASIC.SYSTEM. At the BASIC prompt, I typed "-NUCLEUS", which resulted in the "Cannot find NUCLEUS.DATA : DEAD" error.

Here's the output of a "CAT" command which I executed from the BASIC prompt.
![Microsoft Store 6_29_2022 9_59_07 PM](https://user-images.githubusercontent.com/6971466/176584524-914bb638-7e68-4abb-b965-35bf97263594.png)

(N.B.: Interestingly enough, by deleting BASIC.SYSTEM, renaming NUCLEUS to NUCLEUS.SYSTEM, and booting, all the problems disappear, and the demo works correctly.)

**Comment by ogoguel** [comment]
Understood. I bellieve it is because the filename « Nucleus.system » has been renamed into « Nucleus ». I will update the loader not to depend on the naming. Thanks for your help.

**Comment by robinhood2014** [comment]
Good idea. By the way, all the other methods of launching NUCLEUS.SYSTEM work correctly (e.g. typing "-NUCLEUS.SYSTEM" at the BASIC prompt or launching it from whatever "BYE" screen is bundled with ProDOS), apart from attempting to launch from GS/OS, which is the only issue remaining.
